realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -531,6 +531,7 @@ private static String getManagementRealmUrl(URL authUrl) {
      * @return
      */
     public PermissionManager getPermissionManager() {
+        new AndroidCapabilities().checkCanDeliverNotification("The PermissionManager can only opened on a Looper thread.");
```

Comment:
This is being merged to a feature branch which is still WIP, so no changelog needed (yet)